### DISCUSSION TITLE
【新功能提议】能否增加插入电子版签名以及日期的功能？

### DISCUSSION THREAD

**Discussion by MasterWinston** [BITNP/BIThesis#479] [General]
在研究成果声明和学位论文使用权页面，能否增加插入电子版签名和日期的功能，并可以自动resize插入的电子签名图片，以适应排版。

**Comment by YDX-2147483647** [comment]
本科模板也有这种问题，那边现在的设计是直接导入PDF。

![图片](https://github.com/BITNP/BIThesis/assets/73375426/554afecc-b0fc-41b1-8768-ba5fe742215a)

https://github.com/BITNP/BIThesis/blob/3fd7b2269e8840737d4f82fd1e3a981104b584dc/templates/undergraduate-thesis/main.tex#L105-L114
